Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.py

```diff
@@ -393,6 +394,31 @@ def list_request(self, form: str, query: str = None) -> Dict[str, Any]:
         response = self._http_request("GET", f"arsys/v1/entry/{form}", params=params)
         return response
 
+    def worklog_attachment_get_request(self, worklog_id: str) -> List[fileResult]:
+        """
+        Get BmcITSM Work Log Attachments.
+
+        Args:
+            worklog_id (str): The Work Log ID to pull the attachments from.
+
+        Returns:
+            Dict[str, Any]: API response from BmcITSM.
+        """
+        attachments = []
+        for i in range(1,4):
+            res = self._http_request("GET",
+                                     f"arsys/v1/entry/HPD:WorkLog/{worklog_id}/attach/z2AF Work Log0{i}",
+                                     resp_type="response")
+            try:
+                content_disposition = res.headers['content-disposition']
+            except KeyError:
+                continue
+            fname = re.findall("filename\*?=([^;]+)", content_disposition, flags=re.IGNORECASE)
+            fname = fname[0].strip().strip('"')
```

Comment:
I believe filename will always be there because you can't upload a nameless file as a worklog. All the worklog attachments have a name. If there is a content disposition there is name.